CHANGELOG.md

```diff
@@ -6,19 +6,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [0.2.0] - 2019-09-19
+## [0.3.0] - 2019-12-26
```

Comment:
Should we change the version from `0.2.0` to `0.3.0` or to `0.2.1`?
@InbalZilberman @Tovli  wdyt?

Reply:
@izgeri can you give any insight on this?

Reply:
We usually follow semver for this (https://semver.org/). I'd bump the minor version if you feel like this adds functionality and patch otherwise.

Reply:
thanks, that's very educating.In that case i am fine with the minor version bump.